Mathlib/Data/Int/Bitwise.lean

```diff
@@ -84,6 +84,7 @@ protected def xor : ℤ → ℤ → ℤ
 
 /-- `m <<< n` produces an integer whose binary representation
   is obtained by left-shifting the binary representation of `m` by `n` places -/
+@[deprecated "No replacement." (since := "2025-08-25")]
```

Comment:
I don't think we should ever use "no replacement", here I think we should say "shift by a Nat instead", and if elaboration is really not worth the effort in general, "no replacement, see <PR url>" at least gives the reader a thread to follow